docs/sources/explore/trace-integration.md

```diff
@@ -79,7 +79,7 @@ You can navigate from a span in a trace view directly to logs relevant for that
 
 Click the document icon to open a split view in Explore with the configured data source and query relevant logs for the span.
 
-## Service Graph view
+## Service Graph View
```

Comment:
If it's not capitalized in the text, it shouldn't be capitalized in the heading. Instinct says we shouldn't refer to Service Graph View, but rather Service graph view and Node graph view, though I'm guessing they're capitalized in the UI and that's why you're making these edits..

Reply:
That's a good point. We refer to "Trace View" as the full name of the component displaying trace details. In case of "Node Graph" and "Service Graph" we don't really use "view" as the word "graph" indicates type of visualization. 

Maybe it would make more sense to use: Trace View, Service Graph and Node Graph? Also use capitalized "view" only in "Trace View"?

Reply:
Updated in 0d5dedc7d1e1ec378e256e2cbb6e4306379acc28